### PR TITLE
Updated link text in MHV Landing Page

### PR DESCRIPTION
**Pull request by carlosfelix2**
## Summary
- Updated the text for three links

## Related issue(s)
https://github.com/department-of-veterans-affairs/va.gov-team/issues/82679

## Testing done
Existing tests

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |    localhost_3001_my-health__k=6(iPhone 14 Pro Max) (1)    |   localhost_3001_my-health__k=6(iPhone 14 Pro Max)    |
| Desktop |    localhost_3001_my-health__k=6 (1)    |   localhost_3001_my-health__k=6    |

## What areas of the site does it impact?
- MHV Landing Page

## Acceptance criteria
Update link text as follows:
- Messages
  - Update "Inbox" to read "Go to inbox"
- Appointments
  - Update "Upcoming VA appointments" to read "Manage upcoming appointments"
- Payments
  - Update "File a claim for travel reimbursement" to "File a claim for travel reimbursement (opens in new tab)"

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution
- [x] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

### CODE REVIEW

File: src/applications/mhv-landing-page/utilities/data/index.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ const resolveLandingPageLinks = (
       {
         ...HEALTH_TOOL_LINKS.MESSAGES[0],
         oldHref: mhvUrl(authdWithSSOe, 'secure-messaging'),
-        oldText: 'Inbox',
+        oldText: 'Go to inbox',
```
Comment (radavis) [inline src/applications/mhv-landing-page/utilities/data/index.js:41]: You can omit this line, and the link will have the text from constants.js

**Review by radavis — APPROVED** [review]

**Review by jzucadi — APPROVED** [review]

File: src/applications/mhv-landing-page/utilities/data/index.js (hunk 1 above)
Comment (carlosfelix2) [reply src/applications/mhv-landing-page/utilities/data/index.js:41]: Actually, no. I had to change it here too, otherwise it will read just Inbox.

Comment (radavis) [reply src/applications/mhv-landing-page/utilities/data/index.js:41]: I'm saying if you delete that line, the text _will_ read as "Go to inbox"

Comment (carlosfelix2) [reply src/applications/mhv-landing-page/utilities/data/index.js:41]: Oh, that is very different. Thx

Comment (radavis) [reply src/applications/mhv-landing-page/utilities/data/index.js:41]: Turns out I was completely wrong, here! Sorry!

**Pull request merged**